src/lib/components/navbar/Navbar.component.js

```diff
@@ -120,6 +148,13 @@ function NavBar({
           <ProductNameSpan>{productName}</ProductNameSpan>
         </NavbarMenuItem>
       </NavbarMenu>
+      <NavbarTabs>
+        {tabs.length > 0 &&
+          tabs.map(
+            ({ title }) => <TabItems>{title}</TabItems>
+          )
+        }
+      </NavbarTabs>
       <NavbarMenu>
         {languages.length > 0 && (
```

Comment:
You also might do the same here :ok_woman:

Reply:
I don't agree, it is not the same case here

Reply:
oh, i'm sorry didnt saw the rest of the code, assumed it was the same case